.github/workflows/ci.yml

```diff
@@ -18,8 +18,6 @@ jobs:
           - all
           - debug,default
           - debug,esplora
-          - debug,snap
```

Comment:
Why are we removing `snap` here? Is it intended?

Reply:
hmmm it was on purpose for 2 reasons:
- `snap` seems too specific to MetaMask and I think has no real place in the bdk-wasm . Most likely will be removed.
- `snap` has no tests and is equivalent to `default` . We can't run Snap tests outside of the Snap environment.

Reply:
added back before we decide on removing the `snap` feature.